src/components/OrphanTableRows.tsx

```diff
@@ -1,42 +1,41 @@
 import React from "react";
-import * as Types from "./../utils/Types";
-import * as Renderers from "../utils/Renderers";
 import * as EventHandlers from "../utils/EventHandlers";
+import * as Renderers from "../utils/Renderers";
 import * as StateUpdaters from "../utils/StateUpdaters";
+import * as Types from "./../utils/Types";
 
 const OrphanTableRows = (props: Types.OrphanTableRowsProps): JSX.Element => {
   const [state, dispatch] = props.stateManager;
-  const rows = props.givenCombinedRow.rows;
-  const combinedRowColor = Renderers.getTableRowColor(
-    props.givenCombinedRow,
-    state.currentCombinedRow,
-    props.givenIndex
-  );
-  const markerIdentifier = StateUpdaters.getMarkerIdentifier(
-    props.givenCombinedRow.index
-  );
+  const givenRow = props.givenCombinedRow;
+  const currentRows = state.currentCombinedRows;
+  const markerIdentifier = StateUpdaters.getMarkerIdentifier(givenRow.index);
```

Comment:
Related to "Refactored table rows coloring to have dark odd and light even rows"